src/Aspire.Hosting/ApplicationModel/CustomResourceSnapshot.cs

```diff
@@ -77,6 +82,14 @@ public sealed record EnvironmentVariableSnapshot(string Name, string? Value, boo
 /// <param name="IsInternal">Determines if this url is internal.</param>
 public sealed record UrlSnapshot(string Name, string Url, bool IsInternal);
 
+/// <summary>
+/// A snapshot of a volume, mounted to a container.
+/// </summary>
+/// <param name="Source">The name of the volume. Can be <c>null</c> if the mount is an anonymous volume.</param>
+/// <param name="Target">The target of the mount.</param>
+/// <param name="IsReadOnly">Whether the volume mount is read-only or not.</param>
+public sealed record VolumeSnapshot(string? Source, string Target, bool IsReadOnly);
```

Comment:
I'm wondering whether we should capture the volume type here. You should be able to tell intuitively from the source but might be worth making it obvious whether it is a bind mount or a volume mount.

Reply:
I'll add it to the model. In the UI we currently use name/value pairs in the details view, so won't show it (or `IsReadOnly`) without further work.

Reply:
I found a way to show it: 

![image](https://github.com/user-attachments/assets/fcf80b18-4331-4c9b-83f9-0f83fce684d3)

The visualizer/copy button works with the raw value only (`/var/lib/rabbitmq`), which I think makes the most sense.

This shows both the mount type, and whether the mount is readonly or not. In this case I hard-coded the value to true, as I didn't see an example in the playground.